config/gulp/svg-sprite.js

```diff
@@ -0,0 +1,56 @@
+const gulp = require('gulp');
+const del = require('del');
+const svgSprite = require('gulp-svg-sprite');
+const rename = require('gulp-rename');
+const svgPath = 'src/img';
+
+
+// More complex configuration example
+config = {
```

Comment:
It must already be defined somewhere else, in a global. Or it would be throwing an error. If you want this config to be confined to this closure then you can set it as a const.